sdk/framework/identity_test.go

```diff
@@ -0,0 +1,100 @@
+package framework
+
+import (
+	"testing"
+
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func TestIdentityTemplating(t *testing.T) {
+	sysView := &logical.StaticSystemView{
+		EntityVal: &logical.Entity{
+			ID:   "test-id",
+			Name: "test",
+			Aliases: []*logical.Alias{
+				{
+					ID:            "alias-id",
+					Name:          "test alias",
+					MountAccessor: "test_mount",
+					MountType:     "secret",
+					Metadata: map[string]string{
+						"alias-metadata": "metadata-value",
```

Comment:
Better to use a different value than "metadata-value" in the three places it occurs?  Would reduce ambiguity in test case expected values.